netkan/netkan/status.py

```diff
@@ -108,7 +108,7 @@ def restore_status(cls, filename):
     @classmethod
     def last_indexed_from_git(cls, ckanmeta_repo, identifier):
         try:
-            return parse(ckanmeta_repo.git.log('--', identifier, format='%aI', max_count=1).split("\n")[0])
+            return parse(ckanmeta_repo.git.log('--', identifier, format='%aI', max_count=1).split("\n")[0]).astimezone(timezone.utc)
         except Exception as exc:  # pylint: disable=broad-except
```

Comment:
As noted on @DasSkelett 's PR, the timezone stuff isn't required. PynamoDB's `UTCDateTimeAttribute` takes care of this. But as a once off task, this is NBD.